ui/app/components/clients/history.js

```diff
@@ -1,124 +1,191 @@
 import Component from '@glimmer/component';
```

Comment:
This file was previously named `dashboard.js` from 1.11 designs, reverting to previous name of `history.js` so correlates to route